packages/client/src/views/SysAdmin/Performance/PerformanceHome.tsx

```diff
@@ -409,12 +409,41 @@ class PerformanceHomeComponent extends React.Component<Props, State> {
             </ResponsiveModalContent>
           </ResponsiveModal>
           <LayoutRight>
-            <PerformanceStats
-              registrationOffices={5}
-              totalRegistrars={200}
-              registrarsRatio={2}
-              citizen={50}
-            />
+            <Query
+              query={LOCATION_STATS}
+              variables={{
+                locationId: this.state.selectedLocation
+                  ? this.state.selectedLocation.id
+                  : undefined,
+                populationYear: timeEnd.getFullYear()
```

Comment:
@tahmidrahman-dsi can we make a single query and pass that to wherever it is needed? I mean the ResponsiveModal also need that query. It is just above the LayoutRight tag